.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.4.9
 - 2.5.7
 - 2.6.5
+- 2.7.0
```

Comment:
Can you update latest Ruby versions?

- 2.4.10
- 2.5.8
- 2.6.6
- 2.7.2

Reply:
Sure, and I'll change this PR title to `Add Ruby 2.7.2 and update Ruby versions in travis.yml` .

Reply:
It seems that the update from 2.7.0 to 2.7.2 has been forgotten :-)

Reply:
Oh sorry... I've added it! 😇

Reply:
CI is failing.
https://travis-ci.org/github/esminc/adhoq/jobs/733932214#L243-L248

Can you update from Ruby 2.4.9 to 2.4.10 for exclude matrix as well?
https://github.com/esminc/adhoq/blob/71f9cec0520151f883444a1c0deac5435743a68d/.travis.yml#L32-L35